Xamarin.PropertyEditing/ViewModels/PropertiesViewModel.cs

```diff
@@ -364,6 +359,11 @@ private void UpdateMembers (IObjectEditor[] removedEditors = null, IObjectEditor
 
 			SetNameable (firstNameable);
 			SetCurrentObjectName (name, this.objEditors.Count > 1);
+
+			UpdateProperties (newPropertySet, removedEditors, newEditors);
```

Comment:
Where is the test covering these changes?

Reply:
In my haste, I totally forgot. I'll add them today.

Reply:
This change was unnecessary. Reverted, so unit tests will stay as is.